docs/CDCKudu-sparksink.md

```diff
@@ -0,0 +1,32 @@
+# Change Tracking SQL Server Streaming Source
+
+Description
+-----------
+CDAP Realtime Plugin for Change Data Capture (CDC) in Kudu.
```

Comment:
This plugin takes input from a CDC source and writes the changes to Kudu

Reply:
fixed